### PR TITLE
Update to rust 2018.

### PR DESCRIPTION
**Pull request by feuerste**

### CODE REVIEW

File: xray/src/bin/build_xray_quadtree.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ extern crate quadtree;
 extern crate scoped_pool;
```
Comment (SirVer) [inline xray/src/bin/build_xray_quadtree.rs]: Grep for "extern crate" and remove them? Same for macro_use.

Comment (feuerste) [reply xray/src/bin/build_xray_quadtree.rs]: Nice, done!

**Comment by feuerste** [comment]
ptal @SirVer

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**